test/board_test.exs

```diff
@@ -35,16 +35,16 @@ defmodule BoardTest do
   end
 
   test "To print an empty board in CLI" do
-    board = Board.setup
-    assert capture_io(fn -> Board.show(board) end) ==
-      "\n\n 1 | 2 | 3\n---+---+---\n 4 | 5 | 6\n---+---+---\n 7 | 8 | 9\n\n\n"
+    board = Board.setup("", "")
+    assert capture_io(fn -> Board.show(board, 1) end) ==
+      "\n\n 1 | 2 | 3\n-----------\n 4 | 5 | 6\n-----------\n 7 | 8 | 9\n\n\n"
   end
 
   test "Update the board's visual on a valid moves in CLI" do
-    board = Board.setup
-    moves = %{x: [{0, 0}, {2, 1}, {1, 2}], o: [{1, 0}, {1, 1}, {2, 2}]}
+    board = Board.setup("", "")
```

Comment:
Consider a different name than "setup". This is a very generic name that does not really describe the action of the function. Maybe `Board.new_board`?